<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: transparent;
}

QPushButton:hover {
    color: rgb(20, 20, 20);
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 		80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>341</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom-left-radius: 70px;
border-top-right-radius: 70px;
background-color: rgb(96, 190, 254);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="load">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="icon">
    <iconset resource="load_playlist.qrc">
     <normaloff>:/icons/res/Icons/check-circle.svg</normaloff>:/icons/res/Icons/check-circle.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset resource="load_playlist.qrc">
     <normaloff>:/icons/res/Icons/arrow-left-circle.svg</normaloff>:/icons/res/Icons/arrow-left-circle.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="removeItem">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>41</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="load_playlist.qrc">
     <normaloff>:/icons/res/Icons/minus-circle.svg</normaloff>
     <disabledon>:/icons/res/Icons/minus-square.svg</disabledon>:/icons/res/Icons/minus-circle.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>41</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="load_playlist.qrc">
     <normaloff>:/icons/res/Icons/plus-circle.svg</normaloff>:/icons/res/Icons/plus-circle.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QTableView" name="table">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>321</width>
     <height>341</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="renameItem">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>41</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="load_playlist.qrc">
     <normaloff>:/icons/res/Icons/edit.svg</normaloff>
     <disabledon>:/icons/res/Icons/edit-3.svg</disabledon>:/icons/res/Icons/edit.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="load_playlist.qrc"/>
 </resources>
 <connections/>
</ui>
